src/main/resources/com/uipath/uipathpackage/UiPathPack/help-orchestratorTenant.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Specify the Orchestrator tenant onto which the package(s) will be packed.
```

Comment:
Specify the tenant of the Orchestrator instance from which library dependencies should be restored.